src/views/Card/CardContent.js

```diff
@@ -26,11 +29,13 @@ function CardContent(props) {
     extra,
     header,
     meta,
+    textAlign,
   } = props
 
   const classes = cx(
     className,
     useKeyOnly(extra, 'extra'),
+    useTextAlignProp(textAlign),
     'content',
   )
```

Comment:
As we update almost all `Card` components, lets also move `className` to end, it's one of our style rules. I overlooked this on initial review.

```jsx
cx(
  useKeyOnly(extra, 'extra'),
  useTextAlignProp(textAlign),
   'content',
  className,
)	
```

Reply:
Should i first write `'ui'`, then functions, then other literals and then `className`?